packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/glue/start-crawler-run.ts

```diff
@@ -0,0 +1,56 @@
+import { Construct } from 'constructs';
+import * as iam from '../../../aws-iam';
+import * as sfn from '../../../aws-stepfunctions';
+import { Stack } from '../../../core';
+import { integrationResourceArn } from '../private/task-utils';
+
+/**
+ * Properties for starting an AWS Glue Crawler as a task
+ */
+export interface GlueStartCrawlerRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * Glue crawler name
+   */
+  readonly crawlerName: string;
```

Comment:
I considered a specification that receives `CfnCfnCrawler` instead of `crawlerName`. The crawler name can be obtained from `CfnCrawler.ref`.
Which would be preferable?